README.md

```diff
@@ -297,13 +297,41 @@ with the running machine (via SSH). It supports the following methods:
     Stops or disconnects the system. This stops running boxes or images and disconnects from
     running systems.
 
-  * `run_command(command, *args, options = {})`
-    `run_command(command_and_args, options = {})`
+  * `run(command, *args, options = {})`
+    `run(command_and_args, options = {})`
 
-    Executes a command on the running machine. The execution is implemented
-    using [Cheetah](https://github.com/opensuse/cheetah) and the `run_command`
-    method behaves mostly the same as
-    [`Cheetah.run`](http://rubydoc.info/github/openSUSE/cheetah/master/Cheetah.run).
+    Executes a command on the running machine. A VM::CommandResult instance is
+    returned which can be used with special RSpec matchers to define the
+    expected behavior. Examples:
+
+      # Expect a certain exit code
+      expect(@vm.run("ls -l")).to have_exit_code(0)
```

Comment:
Providing an empty Regexp to allow empty and non-empty output is not that elegant imo and it would duplicate what the `have_stderr` matcher already does. What about `succeed.with_or_without_stderr`?